helm-org.el

```diff
@@ -273,6 +273,73 @@ current heading."
         :truncate-lines helm-org-truncate-lines
         :buffer "*helm org capture templates*"))
 
+;;; Org tag completion
+
+;; Based on code from Anders Johansson posted on 3 Mar 2016 at
+;; <https://groups.google.com/d/msg/emacs-helm/tA6cn6TUdRY/G1S3TIdzBwAJ>
+
+;; FIXME: This function fails with a "Wrong number of arguments" error
+;; if its name starts with "helm"?!  If its name starts with anything
+;; else, it works fine!
+(defun not-helm-org-completing-read-tags (prompt coll pred req initial hist def inh)
+  (if (not (string= "Tags: " prompt))
+      ;; Not a tags prompt.  Use normal completion by calling
+      ;; `org-icompleting-read' again without this function in
+      ;; `helm-completing-read-handlers-alist'
+      (let ((helm-completing-read-handlers-alist (rassq-delete-all
```

Comment:
Hi there, that's a great catch, sorry about that.  

It's interesting that it says that it "often modifies" the original.  I haven't noticed any problems in the...14 months since this commit, and I guess no one else has either.

Which version of Emacs, Org, and Helm are you using?  I wonder if something changed to cause it to modify the alist more "often".